### DISCUSSION TITLE
Welcome to sgpt Discussions!

### DISCUSSION THREAD

**Discussion by tbckr** [tbckr/sgpt#141] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by austengary33** [comment]
Cool project!

Wondering what's the advantage of using this version over the original (https://github.com/TheR1D/shell_gpt)?

**Reply by tbckr** [reply]
Thanks for your interest in the project!

The SGPT tool offers a few advantages over the original shell_gpt:

1. No Python Dependencies: SGPT eliminates the need for Python dependencies, providing a more straightforward installation process and reducing potential compatibility issues.
2. Single Executable: SGPT is packaged as a single executable, making it easy to manage and distribute without worrying about additional files or dependencies. This simplifies integration into various environments.

Feel free to explore SGPT and let me know if you have any more questions!

**Reply by austengary33** [reply]
Thanks for your reply! Keep up the awesome work, love the project!
